mutation_partition_v2.cc

```diff
@@ -319,7 +324,7 @@ stop_iteration mutation_partition_v2::apply_monotonically(const schema& s, mutat
             if (prev_i) {
                 maybe_drop(prev_i);
             }
-            if (preemptible && need_preempt() && i != _rows.end()) {
+            if (need_preempt() && i != _rows.end()) {
```

Comment:
Not thrilled about introducing a virtual call here. I guess it's not too terrible since apply() usually applies a small number of rows.